HHS/Head-Start-TTADP
# [Prod] Allow closing multi-recipient goals from admin

## Description of change

- Allow closing of multi-recipient goals from admin
- Also make small changes to communication log (results & next steps no longer required)

https://github.com/HHS/Head-Start-TTADP/assets/3288586/fd530ea2-fe0f-4bcf-a47d-d677a03028d8

## How to test

- Confirm that you can close a goal through the admin
- Confirm that the results and next steps fields aren't required on the communication log


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2110
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2379

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
